nix/default.nix

```diff
@@ -113,6 +116,11 @@ in
         }'
       '';
 
+      postInstall = ''
+        wrapProgram $out/bin/Hyprland \
```

Comment:
Maybe it would be better to separate the wrapping into a different derivation. So:

`hyprland-unwrapped` - regular hyprland
`hyprland` - wrapped hyprland

This way people who don't want to use the wrapper or want to provide their own, will be able to opt out of it.

many packages do that in nixpkgs: https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=-unwrapped

Reply:
Will do

Reply:
The current impl looks good to me. Users can still override hyprland as usual, and only those who want to wrap it themselves have to use an extra override option.